src/toil/jobStores/abstractJobStore.py

```diff
@@ -1872,15 +1874,28 @@ class JobStoreSupport(AbstractJobStore, metaclass=ABCMeta):
     stores.
     """
 
+    ftp = None
+
+    @classmethod
+    def _setup_ftp(cls) -> None:
```

Comment:
It's weird that the `ftp` member is public, but the function that must be called before it is actually used has an underscore prefix.